main.go

```diff
@@ -155,47 +158,47 @@ func Initialize(e *echo.Echo, api APIInterface, apiConfig string) *echo.Echo {
 						switch method {
 						case "GET":
 							if !weosConfig.DisableCors {
-								middlewares = append(middlewares, EnableCORS(method, config.BasePath+echoPath))
+								e.Use(EnableCORS(method))
```

Comment:
This looks like it wouldn't work. You're applying the middelware globally instead of to a specific path. The enableCORs also doesn't take a path so I'm not sure I understand how you're restricting the relevant access to specific paths. I think you would get a false positive because your API will cover all the methods and thus it will appear to work